### ISSUE TITLE
Support for logitech wingman extreme digital 3d

### ISSUE THREAD

**Issue by fgma** [necroware/gameport-adapter#6]
I've read in a forum post you are working on this. Any news on this?

**Comment by necroware** [comment]
I implemented the initial support for the Logitech devices. It is currently only tested with the Logitech Wingman Extreme Digital joystick, but hopefully it will also work with other Logitech devices as well. Unfortunately I have no other devices....

Currently the implementation is ready to be tested on the #7 branch

**Comment by fgma** [comment]
Perfect! I will try to test this during the next few days.

**Comment by fgma** [comment]
It works for me. My joystick just has some issues with the Y-Axis range an generally a lot of dead motion in the center. Don't know if it is just my particular hardware or if it is a general problem.

**Comment by necroware** [comment]
That's good news, since the protocol is used universally for all the digital Logitech devices and I was not sure, if my implementation will work with something else, than the Wingman Extreme Digital. Regarding your Y-Axis issues, does it only happen with that one? Are the other axes (X, Throttle) alright? As I got my (used) Wingman Extreme Digital the X-Axis didn't work at all, it jumped wildly left and right. I had to open the Joystick and spray some dioxide on the potentiometers. May be it's the same issue with your device. What about buttons and hats, do they work properly on your Model?
